### DISCUSSION TITLE
Sticky Sliders for Correlation Data

### DISCUSSION THREAD

**Discussion by S-Majoros** [overture-stack/docs#17] [Support] [answered]
Hello! On the correlation tab of the Drug Discovery Data Portal I am unable to move the sliders for Spearman Correlation or Pearson Correlation Square.

**Comment by justincorrigible** [comment, marked as answer]
Sounds like an issue with the way Arranger is rendering the data for that type of value.
Will put it in my ToDos list (aka "tickets"), prioritized for the first couple weeks of 2025 🤞
